charts/astronomer/values.yaml

```diff
@@ -5,7 +5,7 @@
 # This version number controls the default Airflow chart version that will be installed
 # when creating a new deployment in the system. This is also used to ensure all
 # child airflow deployments are kept up to date and on the latest version.
-airflowChartVersion: 0.20.1
+airflowChartVersion: 0.21.0
```

Comment:
I dont think we want to bump to 0.21.0. It should be a 1.0.0* version of airflow chart.